moj-node/server/services/hubContent.js

```diff
@@ -11,6 +11,12 @@ module.exports = function createHubContentService({
     }
 
     const content = await contentRepository.contentFor(id);
+    const prisonId = prop('establishmentId', content);
```

Comment:
This just moves the access check to before it attempts to get suggested Content, no point to get it if we can't get the main content.